src/app/results/results.component.html

```diff
@@ -105,9 +105,6 @@
               </ul>
             </div>
           </div>
-            <div class="result message-bar" *ngIf="totalNumber > 0">
-                {{message}}
-            </div>
```

Comment:
Why you removed `About ... results` from videos section? I think Google displays it.